perf.groovy

```diff
@@ -691,6 +691,109 @@ parallel(
     }
 }
 
+// Setup size-on-disk test
+[true, false].each { isPR ->
+    ['Windows_NT'].each { os ->
+        ['x64', 'x86'].each { arch ->
+            def architecture = arch
+            def newJob = job(Utilities.getFullJobName(project, "sizeondisk_${arch}", isPR)) {
+
+                // Set the label.
+                label('20170427-elevated')
+                wrappers {
+                    credentialsBinding {
+                        string('BV_UPLOAD_SAS_TOKEN', 'CoreCLR Perf BenchView Sas')
+                    }
+                }
+
+                if (isPR) {
+                    parameters {
+                        stringParam('BenchviewCommitName', '\${ghprbPullTitle}', 'The name that you will be used to build the full title of a run in Benchview.  The final name will be of the form <branch> private BenchviewCommitName')
+                    }
+                }
+
+                def channel = 'release/2.0.0'
```

Comment:
We should be running this on master as well as release/2.0.0

Reply:
That is the ideal. The reason it's only release right now is that master breaks many of the templates due to referencing unpublished packages. This is one thing I need to discuss with you tomorrow

Reply:
Are they not even published on Myget?  That seems like it should not be the case.

Reply:
I believe that to be the case. Maybe I am looking at it wrong

Reply:
OK, we can work through it tomorrow.  That surprises me, but I could believe that it is the case.